### PR TITLE
Allow multiple boot partition binaries

### PR DESCRIPTION
**Pull request by landerweit-phytec**

### CODE REVIEW

**Review by mschwan-phytec — CHANGES_REQUESTED** [review]
Code changes look good! One issue with the documentation below.

File: doc/layout-config-reference.rst [hunk 1]
```diff
@@ -116,14 +116,17 @@ eMMC's special boot partitions can be specified using the keyword
...
+
 ``input-offset`` (integer/string)
    Offset of the input data to be written. This keyword is optional.
 
 ``output-offset`` (integer/string)
    Offset of the written output data. This keyword is optional.
 
 ``input`` (mapping)
-   An input mapping. See :ref:`input-files`. This keyword is optional.
+   An input mapping. See :ref:`input-files`.
```
[the remark this answers is not in the document]
Comment (landerweit-phytec) [reply doc/layout-config-reference.rst:129]: Thanks for the suggestion. I've squashed it into my documentation changes.

**Review by mschwan-phytec — APPROVED** [review]
Nice! :slightly_smiling_face: 

Just a hint to keep in mind for next time: If you squash commits from other people consider adding them to the corresponding commit with "Co-developed-by", as describe in https://www.kernel.org/doc/html/latest/process/submitting-patches.html#when-to-use-acked-by-cc-and-co-developed-by

**Comment by landerweit-phytec** [comment]
> Nice! 🙂
> 
> Just a hint to keep in mind for next time: If you squash commits from other people consider adding them to the corresponding commit with "Co-developed-by", as describe in https://www.kernel.org/doc/html/latest/process/submitting-patches.html#when-to-use-acked-by-cc-and-co-developed-by

Added your Co-developed-by

**Comment by mschwan-phytec** [comment]
> > Nice! 🙂
> > Just a hint to keep in mind for next time: If you squash commits from other people consider adding them to the corresponding commit with "Co-developed-by", as describe in https://www.kernel.org/doc/html/latest/process/submitting-patches.html#when-to-use-acked-by-cc-and-co-developed-by
> 
> Added your Co-developed-by

Thanks! :slightly_smiling_face:

**Pull request merged**